modules/serverless-overriding-pdbs-serving.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * /serverless/serving/ha-replicas-serving.adoc
+
+
+:_content-type: PROCEDURE
+[id="serverless-overriding-poddistruptionbudgets-serving_{context}"]
+= Overriding podDisruptionBudgets
+
+A Pod Disruption Budget (PDB) is a standard feature of Kubernetes APIs that helps to limit the disruption to an application when its pods need to be rescheduled for maintenance reasons.
+
+.Procedure
+
+You can override the default PDB for a specific resource by modifying the `minAvailable` configuration value in the `KnativeServing CR`.
+
+[NOTE]
+====
+If you disable high-availability, for example, by changing the `high-availability.replicas` value to 1, make sure you also update the corresponding podDisruptionBudget `minAvailable` value to 0. Otherwise, the podDisruptionBudget will prevent automatic cluster or operator upgrades.
+====
+
+.An example of minAvailable set to 70% for PDB named activator-pdb
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
+kind: KnativeServing
+metadata:
+ name: knative-serving
+ namespace: knative-serving
+spec:
+ podDisruptionBudgets:
+   -    name: activator-pdb
```

Comment:
This formatting is strange, let's use the same as for KnativeEventing. It seems to render correctly though.